src/cdfmean.f90

```diff
@@ -115,10 +124,15 @@ PROGRAM cdfmean
      PRINT *,'                   partial steps.'
      PRINT *,'       [ -var ]  : also compute the spatial variance of cdfvar '
      PRINT *,'       [ -zeromean ] : create a file with cdfvar having a zero spatial mean.'
+     PRINT *,'       [ -out_suff OUT-file-suffix ] : specify SUFFIX (+ directory) of the output files'
+     PRINT *,'       [ -basins n_basins basin_names.. ] : specify basins in ',TRIM(cn_fbasins),' to be used'
      PRINT *,'      '
      PRINT *,'     REQUIRED FILES :'
      PRINT *,'       Files ', TRIM(cn_fhgr),', ', TRIM(cn_fzgr),', ', TRIM(cn_fmsk)
      PRINT *,'      '
+     PRINT *,'     OPTIONAL FILES :'
+     PRINT *,'       File ',TRIM(cn_fbasins),'; if not available only the global domain is calculated for'
+     PRINT *,'      '
```

Comment:
I suggest to change the comment by "File ',TRIM(cn_fbasins),' if -basins or -basins_default activated" because, if cn_bassin is missing and option -basins activates the tools stop (l176) and nothing is done.